CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+**This is a work in progress. Sections will be added over time.**
+
+## contributions
+
+All contributions are welcome. By contributing, you accept to give the
+ownership of your contribution to the [World Wide Web
+Consortium](http://www.w3.org). They are a _nonprofit organization_
+and just want to simplify the governance of this opensource project.
+
+
+## style guide
+
+Your code must look like that:
+
+```scala
+package org.w3.banana
+package mypackage
+
+import scala.util.{ Try, Success, Failure }
+import java.net.{ URI => jURI, _ }
+// do not be afraid of wildcards
+import java.io._
+
+/**
+ * First sentence ending with a period.
+ *
+ * Then next paragraph.
+ * And a second line.
+ *
+ * And so on.
+ */
+class Foo[Rdf <: RDF](
+  foo: String,
+  bar: String
+)(implicit
+  ops: RDFOps[Rdf],
+  writerSelector: RDFWriterSelector[Rdf],
+  turtleReader: RDFReader[Rdf, Turtle],
+  turtleWriter: RDFWriter[Rdf, Turtle],
+  jsonldReader: RDFReader[Rdf, JsonLdCompacted],
+  jsonldWriter: RDFWriter[Rdf, JsonLdCompacted]
+) extends WordSpec with Matchers {
+
+  val longList = List(
+    1,
+    "foo"
+  )
+
+  val list: List[Any] = List(1, "foo")
+
+  def baz(toto: Int, titi: String = "bazinga")(implicit ev: Show[Int]): Unit = {
+    ???
+  }
+
+  // that's an inline comment helpful when reading the code
+  // there could be several lines too!
+  // TODO no really, do it some day
+  list
+    .filter(! _.isEmpty)
+    .map(s => s.toString.toUppercase)
+    .flatMap { s =>
+      ???
+    }
+
```

Comment:
Why only one then? What about 2? Here the rule is simple: each function gets a new line.